apis/v1alpha1/gateway_types.go

```diff
@@ -292,6 +294,26 @@ type Listener struct {
 	// are compatible with the application protocol specified in
 	// the Protocol field.
 	//
+	// Although a client request may technically match multiple route rules,
+	// only one may ultimately receive the request. Matching precedence MUST be
+	// determined in order of the following criteria:
+	//
+	// * The most specific match. For example, the most specific HTTPRoute match
+	//   would be determined by the longest matching combination of hostname and
+	//   path.
+	// * The oldest Route based on creation timestamp. For example, a Route with
+	//   a creation timestamp of "2020-09-08 01:02:03" would be given precedence
+	//   over a Route with a creation timestamp of "2020-09-08 01:02:04".
+	// * The Route appearing first in alphabetical order (namespace/name). For
+	//   example, foo/bar would be given precedence over foo/baz.
+	//
+	// All valid portions of the Routes selected by this field should be
```

Comment:
s/the Routes/a Route/ to match the following sentence > `Invalid portions of a Route`.

Reply:
Updated, thanks!